data/campaigns/tutorial/scenarios/02_Tutorial_part_2.cfg

```diff
@@ -869,6 +869,24 @@ A full list of abilities and weapons specials, along with traits, may be found i
         [/filter_condition]
 
         {TALK_ABOUT Thrag ( _ "You are close to killing their leader! The unit who finishes him will gain 16 experience points because he is second level. Choose your attacking unit carefully!")}
+
+        [message]
+            speaker=student
+            message= _ "Should I choose a unit that will level up?"
+            message_female= _ "female^Should I choose a unit that will level up?"
+        [/message]
+
+        {TALK_ABOUT Thrag ( _ "That’s a difficult question. Consider which advancements will strengthen your army, and whether the unit should advance now or later.")}
+
+        {TALK_ABOUT Thrag ( _ "Some second level units strengthen your army far more than just having a stronger version of the first level unit. A Captain’s leadership makes every level 1 unit do 25% more damage when he’s next to them. A Druid heals 8 hitpoints instead of 4, and her ability to cure poison is invaluable when you are facing enemies that use it. The powerful, accurate attacks of a Sorceress or Marksman are good for dislodging fortified enemy units. Having these units already levelled on your recall list can make a huge difference in the next battle.")}
```

Comment:
```suggestion
        {TALK_ABOUT Thrag ( _ "Many level 2 units gain unique abilities and specials that make them more than simple upgrades to their level 1 counterparts. A Captain’s leadership makes every adjacent level 1 unit do 25% more damage. A Druid heals 8 hitpoints instead of 4 and gains the ability to cure poison. The powerful, accurate attacks of a Sorceress or Marksman are good for dislodging fortified enemy units. Having these units already leveled on your recall list can make a huge difference in future battles.")}
```

Reply:
Thanks for rewording this.
* I'll take your wording for the Captain
* I want to keep my first sentence, to focus on the strength of the army, not the individual unit
* I'll keep my version about the Druid, both because the player hasn't met any units with poison yet, and because it's about why that unit helps more in some battles than in others